--- /usr/lib/python3.13/site-packages/koji/rpmdiff.py   2025-03-18 01:00:00.000000000 +0100
+++ /usr/lib/python3.13/site-packages/koji/rpmdiff.py   2025-03-25 14:00:05.695590906 +0100
@@ -167,6 +167,7 @@
     # load a package from a file or from the installed ones
     def __load_pkg(self, filename):
         ts = rpm.ts()
+        ts.setVSFlags(rpm._RPMVSF_NOSIGNATURES)
         f = os.open(filename, os.O_RDONLY)
         hdr = ts.hdrFromFdno(f)
         os.close(f)
